service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpConnectionHandler.java

```diff
@@ -14,45 +16,64 @@
 import com.microsoft.azure.sdk.iot.service.IotHubServiceClientProtocol;
 import com.microsoft.azure.sdk.iot.service.ProxyOptions;
 import com.microsoft.azure.sdk.iot.service.Tools;
+import com.microsoft.azure.sdk.iot.service.exceptions.IotHubException;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.qpid.proton.Proton;
-import org.apache.qpid.proton.engine.*;
+import org.apache.qpid.proton.engine.Connection;
+import org.apache.qpid.proton.engine.Event;
+import org.apache.qpid.proton.engine.Session;
+import org.apache.qpid.proton.engine.SslDomain;
+import org.apache.qpid.proton.engine.Transport;
 import org.apache.qpid.proton.engine.impl.TransportInternal;
+import org.apache.qpid.proton.reactor.FlowController;
+import org.apache.qpid.proton.reactor.Handshaker;
 import org.apache.qpid.proton.reactor.Reactor;
 
 import javax.net.ssl.SSLContext;
 import java.io.IOException;
 
 @Slf4j
-public class AmqpConnectionHandler extends ErrorLoggingBaseHandlerWithCleanup
+public abstract class AmqpConnectionHandler extends ErrorLoggingBaseHandlerWithCleanup implements CbsSessionStateCallback
 {
-    public static final String WEBSOCKET_PATH = "/$iothub/websocket";
-    public static final String WEBSOCKET_SUB_PROTOCOL = "AMQPWSB10";
-    public static final int AMQPS_PORT = 5671;
-    public static final int AMQPS_WS_PORT = 443;
+    private static final String WEBSOCKET_PATH = "/$iothub/websocket";
+    private static final String WEBSOCKET_SUB_PROTOCOL = "AMQPWSB10";
+    private static final int AMQPS_PORT = 5671;
+    private static final int AMQPS_WS_PORT = 443;
```

Comment:
might not be high impact, but we'll need to call this out as "breaking" in our release notes, right? (changing visibility of public fields)

Reply:
This class isn't considered part of our API surface, so we should be fine here.